_sass/_jumbotron.scss

```diff
@@ -23,13 +23,18 @@
 .jumbotron .logoimg {
   text-align: center;
   padding-bottom: 3rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: center;
 }
 
 .jumbotron .logoimg img {
 
   @media screen and (max-width: 993px) {
-    width: 70%;
+    height: 60px;
```

Comment:
@AndreiZiatiev why did you change this line, please add a reason message here.

Reply:
form me ot looks prettier